### ISSUE TITLE
[Facebook] Error downloading Facebook videos

### ISSUE THREAD

**Issue by rhenanbartels** [yt-dlp/yt-dlp#7994]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

yt-dlp is raising `ExtractorError('Cannot parse data')` error when trying to download facebook videos.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.facebook.com/watch?v=508246001379256']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.9.14 (CPython x86_64 64bit) - Linux-5.15.0-1039-aws-x86_64-with-glibc2.31 (OpenSSL 1.1.1n  15 Mar 2022, glibc 2.31)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.07.22, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[facebook] Extracting URL: https://www.facebook.com/watch?v=508246001379256
[facebook] 508246001379256: Downloading webpage
[facebook] 508246001379256: Downloading webpage
ERROR: [facebook] 508246001379256: Cannot parse data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "/usr/src/.venv/lib/python3.9/site-packages/yt_dlp/extractor/common.py", line 710, in extract
    ie_result = self._real_extract(url)
  File "/usr/src/.venv/lib/python3.9/site-packages/yt_dlp/extractor/facebook.py", line 707, in _real_extract
    return self._extract_from_url(real_url, video_id)
  File "/usr/src/.venv/lib/python3.9/site-packages/yt_dlp/extractor/facebook.py", line 654, in _extract_from_url
    raise ExtractorError('Cannot parse data')

```
```

**Comment by rhenanbartels** [comment]
Closed in favor of https://github.com/yt-dlp/yt-dlp/issues/7901

**Comment by japhenchen** [comment]
same ..... 

 yt-dlp https://fb.watch/mPJ5Ngsz_p/ -vU
[debug] Command-line config: ['https://fb.watch/mPJ5Ngsz_p/', '-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.10.12 (CPython x86_64 64bit) - Linux-5.15.90.4-microsoft-standard-WSL2-x86_64-with-glibc2.35 (OpenSSL 3.0.2 15 Mar 2022, glibc 2.35)
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, secretstorage-3.3.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[generic] Extracting URL: https://fb.watch/mPJ5Ngsz_p/
[generic] mPJ5Ngsz_p: Downloading webpage
[redirect] Following redirect to https://www.facebook.com/mlbcts/videos/187864833931070/
[facebook] Extracting URL: https://www.facebook.com/mlbcts/videos/187864833931070/
[facebook] 187864833931070: Downloading webpage
[facebook] 187864833931070: Downloading webpage
ERROR: [facebook] 187864833931070: Cannot parse data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "/usr/local/lib/python3.10/dist-packages/yt_dlp/extractor/common.py", line 710, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.10/dist-packages/yt_dlp/extractor/facebook.py", line 707, in _real_extract
    return self._extract_from_url(real_url, video_id)
  File "/usr/local/lib/python3.10/dist-packages/yt_dlp/extractor/facebook.py", line 654, in _extract_from_url
    raise ExtractorError('Cannot parse data')
